src/pages/Subscription/Subscription.jsx

```diff
@@ -0,0 +1,15 @@
+
+export default function SubscriptionPage(){
+	return (
+		<div>

```

Comment:
You can use `<Fragment>` instead of an empty div.

Fragment coming from:
```js
import { Fragment } from 'react';
```

And you can use like a component:
```tsx
<Fragment>
  <div className="info">
    <h1>PING.</h1>
    <p>We are lauching soon!</p>
    <p>Subscribe and get notified</p>
  </div>
  <div className="subscription">
    <input type="text" />
  </div>
</Fragment>
```

Reply:
Changed the empty div to Fragment.